crates/primitives/kdf/Readme.md

```diff
@@ -0,0 +1,8 @@
+# KDF
+
+This crate provides an interface to the key derivation functions (KDFs) implemented in libcrux.
+
+We currently support:
+
+* Extract-then-Expand: HKDF
+* PRF: HMAC, Blake2
```

Comment:
Not sure if people should really use hmac as a prf. But there are of course some historical protocols that do.